### DISCUSSION TITLE
Удаление из плейлиста треков другого собственного плейлиста

### DISCUSSION THREAD

**Discussion by sergvpr** [Chimildic/YaMuTools#5] [Идея] [answered]
Кейс использования - удаление задублированных треков в разных плейлистах, определение только залайканых треков, отсутствующих в плейлисте/-ах.
По первому вроде просто, а во втором случае создаем плейлист с треками из плейлиста, например, "Мне нравится", и удаляем треки имеющиеся в других плейлистах, останутся треки с лайками, которых нет в других местах, можно будет их перераспределить.

**Comment by sna077791** [comment]
Поддерживаю. В том же "Плейлист дня" постоянно есть треки из моих плейлистов((( Хотелось бы такую очистку в будущем.

**Comment by Chimildic** [comment, marked as answer]
В версии `0.7.2` появилась возможность вычитать треки из текущего плейлиста по образцу других выбранных плейлистов.
Обновление проходит проверку в Web Store.

**Reply by sergvpr** [reply]
Работает чётко. Проверял много раз.
